Src/Common.props

```diff
@@ -54,7 +54,7 @@
     <DefineConstants Condition=" '$(TargetFramework)'=='netstandard2.0' Or '$(TargetFramework)'=='netcoreapp2.1' Or '$(TargetFramework)'=='net6.0' ">$(DefineConstants);SYSTEM_NET_MAIL</DefineConstants>
   </PropertyGroup>
   <PropertyGroup>
-    <DefineConstants Condition=" '$(TargetFramework)'=='net6.0' ">$(DefineConstants);SYSTEM_DATEONLY</DefineConstants>
+    <DefineConstants Condition=" '$(TargetFramework)'=='net6.0' ">$(DefineConstants);NET6_0_OR_GREATER</DefineConstants>
```

Comment:
No need to define this yourself. the SDK will take care of defining these symbols

https://docs.microsoft.com/en-us/dotnet/standard/frameworks#how-to-specify-a-target-framework

Reply:
Thank you, I was unaware the SDK did the `_OR_GREATER` constants. Changed.

Reply:
We just need to make sure that we use a compatible .NET SDK in the CI pipeline

@aivascu